tests/integration/query/simple/with_order_test.go

```diff
@@ -827,7 +814,6 @@ func TestQuerySimple_WithInvalidOrderEnum_ReturnsError(t *testing.T) {
 func TestQuerySimple_WithMultipleOrderFields_ReturnsError(t *testing.T) {
 	tests := []testUtils.TestCase{
 		{
-			Description: "Simple query with multiple order fields and a single entry",
```

Comment:
todo: I do think this test, and all other tests that are actually multiple tests squashed into one  test function, need to be split up before we can remove `Description`.  Without the description it becomes even harder to tell which test failed, and it is already hard enough.

Happy enough to argue on this though, as often when one of them fails I find it least difficult to delete/comment out tests until I figure out which one is failing - IIRC some of the have the same description anyway, and sometimes the logs are noisy/unclear enough to make test-deletion easier than trying to read the output.

Reply:
I like this thought, but isn't it a different issue to address?

Reply:
- [x] Done in the commit: ` PR(FIX): Split all multiple tests`